README.md

```diff
@@ -10,6 +10,9 @@ Introduction: ONNXRuntime-Extensions is a C/C++ library that extends the capabil
 ## Quickstart
 The library can be utilized as either a C/C++ library or other advance language packages like Python, Java, C#, etc. To build it as a shared library, you can use the `build.bat` or `build.sh` scripts located in the root folder. The CMake build definition is available in the `CMakeLists.txt` file and can be modified by appending options to `build.bat` or `build.sh`, such as `build.bat -DOCOS_BUILD_SHARED_LIB=OFF`. For more details, please refer to the [C API documentation](./docs/c_api.md).
 
+### Android 16KB Page Size Support
```

Comment:
this is good information, but I wonder if it belongs in the top-level readme. maybe it would be better as a code comment.